src/split.c

```diff
@@ -119,8 +119,23 @@ void split_child_worker(proc_arg_t args,
                 if(lossy==0){
                     slow5_aux_meta_t* aux_ptr = slow5File_i->header->aux_meta;
                     uint32_t num_aux_attrs = aux_ptr->num;
+                    int aux_add_fail = 0;
                     for(uint32_t r=0; r<num_aux_attrs; r++){
-                        if(slow5_aux_meta_add(slow5File->header->aux_meta, aux_ptr->attrs[r], aux_ptr->types[r])){
+                        if(aux_ptr->types[r]==SLOW5_ENUM || aux_ptr->types[r]==SLOW5_ENUM_ARRAY){
```

Comment:
@sashajenner also this is how an aux_map is copied to a new slow5 file. Here, I create many small slow5 files from a big one.

Reply:
looks fine